packages/flutter_tools/schema/pubspec_yaml_android.json

```diff
@@ -0,0 +1,214 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "androidApp": {
+      "appTitle": { "type": "string" },
+      "appVersionName": { "type": "string" },
+      "appVersionCode": { "type": "string" },
+      // packageName is same as androidPackage, but seems like the answer now is
```

Comment:
Comments should be avoided in JSON - most parsers won't handle them. Maybe move to `"description"` in the `"packageName"` object?

Reply:
I can find a better comment solution once we reach consensus on the schema itself.  I figured for now it's best to include important links nearby for PR reference.  But yeah, gotta do something different with those.

Reply:
FYI, https://github.com/Workiva/json_schema/issues/30 (not available yet, unfortunately)